js-miniapp-sample/src/pages/user-details.js

```diff
@@ -157,40 +157,56 @@ export const dataFetchReducer = (state: State, action: Action) => {
         isLoading: true,
         isError: false,
         hasRequestedPermissions: false,
+        isPointsLoading: false,
+        isPointsError: false,
+        hasRequestedPointPermissions: false,
```

Comment:
I think it's incorrect to add all of the state items here: this will reset state items which the action had no affect on. For example, if points is already loading or has an error, then this will incorrectly reset it to `false`. 
The `...state` line above is already picking the correct values for these state items.

Reply:
Ok, I have updated the PR to reset only the permission item state during INIT & SUCCESS operations.